CODE_OF_CONDUCT_fr.md

```diff
@@ -0,0 +1,52 @@
+# Code de Conduite Contributeurs
+
+En tant que contributeurs et responsables de ce projet, et dans l'intérêt
+de favoriser une communauté ouverte et accueillante, nous nous engageons à
+respecter toutes les personnes qui contribuent en rapportant des erreurs,
+postant des demandes de fonctionnalités nouvelles, mettant à jour la
+documentation, soumettant des _pull requests_ ou des _patches_, ainsi que
+toute autres activité.
+
+Nous sommes déterminés à faire de la participation dans ce projet une
```

Comment:
That sentence is a bit weird to me. I would ratter say "Nous sommes déterminés à rendre toute participation à ce projet une expérience [...]"